src/Components/home/sectionFour/SectionFour.js

```diff
@@ -0,0 +1,81 @@
+                            <span className='w-[50px] h-[2px] bg-red-500 inline-block'></span>
+                            Featured Category
+                        </h1>
+                        <p className='py-3 text-[#4A5355]'>
+                            Architect client-centered total linkage for intuitive benefits. Dynamically restore convergence before real-time partnerships Dynamically restore convergence before real-time.
+                        </p>
+                    </div>
+                </div>
+
+                <div className='grid grid-cols-[1fr_404px] max-2xl:grid-cols-[1fr_370px] max-lg:grid-cols-1 gap-4 mt-[30px] relative'>
+                    <div>
+                        <img data-aos="fade-right" data-aos-delay="400"
+                            className='rounded-xl'
+                            src='https://eduquest.itech-theme.com/wp-content/uploads/2023/08/service-1.jpg' />
+                        <div
+                            data-aos="flip-up" data-aos-delay="500"
+                            className='max-w-[723px] py-[35px] px-[30px] bg-white shadow-lg relative bottom-16 left-1'>
+                            <h1 className='text-[#002935] text-[36px] font-extrabold mb-5'>
+                                Digital Marketing</h1>
+                            <p className='mb-5 text-[#4A5355] text-[18px] font-normal'>
+                                Architect client-centered total linkage for intuitive benefits. Dynamically restore convergence before convergence before real-time .
+                            </p>
+                            <Link href={'#'} className='flex items-center gap-2 text-[#FF3158] font-bold text-[16px]'>
+                                Explore more
+                                <IoIosArrowRoundForward size={20}/>
+                            </Link>
+                        </div>
+                    </div>
+                    
+                    <div data-aos="flip-right" data-aos-delay="500" className=' pt-[40px] z-0 relative h-fit bg-[#002935] text-white rounded-xl max-lg:bottom-10'>
+                        <div className='text-[28px] font-extrabold mb-[20px] px-[20px]'>
+                            <span className='w-5 h-5 rounded-full inline-block bg-red-500 mr-[15px]'></span>
+                            Explore The Future
+                        </div>
+                        <ul className='text-[18px] font-extrabold divide-y-[1px] rounded-xl pb-[20px] divide-[#1a434f] bg-[#002935] px-[20px]'>
+                            <List href='#' title='Development'/>
+                            <List href='#' title='Digital Marketing'/>
+                            <List href='#' title='Financial'/>
+                            <List href='#' title='Business Consulting'/>
+                            <List href='#' title='Graphic design'/>
+                            <List href='#' title='Development'/>
+                        </ul>
```

Comment:
under <ul> it should be <li> but in your component there are no declare about li.